features/gba-dev/Informatievragen-fields-gba.feature

```diff
@@ -0,0 +1,634 @@
+      | burgerservicenummer | 000000255                       |
+      | fields              | leeftijd                        |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam             | waarde   |
+      | geboorte.datum   | 19960229 |
+      | overlijden.datum | 20151001 |
+
+  Rule: wanneer gevraagd wordt om indicatieOverleden, wordt overlijden.datum geleverd
+
+    Scenario: met fields is gevraagd om indicatieOverleden van een overleden persoon
+      Gegeven de persoon met burgerservicenummer '000000267' heeft de volgende 'overlijden' gegevens
+      | naam                     | waarde        |
+      | datum overlijden (08.10)  | 20151001     |
+      | plaats overlijden (08.20) | Buenos Aires |
+      | land overlijden (08.30)   | 7015         |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000267                       |
+      | fields              | indicatieOverleden              |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam  | waarde   |
+      | overlijden.datum | 20151001 |
+
+  Rule: wanneer een veld van verblijfplaats wordt gevraagd, worden de velden geleverd die nodig zijn voor het bepalen van het type verblijfplaats
+    - dit betreft alle velden uit groepen 11 (adres), 12 (locatie) en land, regel 1, regel 2 en regel 3 van groep 13 (verblijf buitenland)
+
+    Abstract Scenario: met fields wordt gevraagd naar een in een <adressoort> <beschikbaarheid> veld 
+      Gegeven de persoon met burgerservicenummer '<burgerservicenummer>' heeft de volgende 'verblijfplaats' gegevens
+      | naam                                   | waarde   |
+      | gemeente van inschrijving (09.10)      | 0599     |
+      | functieAdres.code (10.10)              | W        |
+      | datum aanvang adreshouding (10.30)     | 20150808 |
+      En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+      | naam                                       | waarde                             |
+      | straatnaam (11.10)                         | <straat>                           |
+      | naam openbare ruimte (11.15)               | <straat>                           |
+      | huisnummer (11.20)                         | <huisnummer>                       |
+      | postcode (11.60)                           | <postcode>                         |
+      | woonplaats (11.70)                         | <woonplaats>                       |
+      | identificatiecode verblijfplaats (11.80)   | <adresseerbaarObjectIdentificatie> |
+      | identificatiecode nummeraanduiding (11.90) | <nummeraanduidingIdentificatie>    |
+      | locatiebeschrijving (12.10)                | <locatiebeschrijving>              |
+	  | gemeentecode (92.10)                       | 0599                               |
```

Comment:
92.10 is het elementnummer waarmee de code uit de gemeentetabel (33) bedoeld wordt. 

In andere features is hier gemeente_code voor gebruikt. Reden daarvoor is dat in de groep adres geen gemeente(code) is opgenomen, maar in de lo3_adres tabel wel. Daardoor is dat element van de tabel niet te koppelen aan een elementnummer uit het LO-GBA. 
Mij lijkt de oplossing om gemeente_code zonder elementnummer te gebruiken het beste te passen.